.github/ISSUE_TEMPLATE/bugs_features.md

```diff
@@ -0,0 +1,30 @@
+<!--
+Thanks for your contribution!
+Please fill out the necessary sections, and delete unused ones.
+-->
+
+## Type of Issue: **Bug report|Feature request|Code refactor|Documentation change**
+
+## Steps to Reproduce:
+- one change.
+- another.
+
+## Expected Behavior:
+- one.
+- another.
+
+## Current Behavior:
+- one change.
+- another change.
+
+## Screenshots/ Code Snippets:
+1. one.
+2. ```another.```
+
+## Related URLs:
+e.g. `127.0.0.1:8000/harvest/`
+
+## Checklist:
+- [ ] My development environment is properly setup according to the [installation guide](https://github.com/LesFruitsDefendus/saskatoon-ng/blob/develop/INSTALL.md).
+- [ ] I'm using the latest version of the project.
+- I can debug and fix this issue: YES|NO
```

Comment:
```suggestion
```

The issue tracker might also be used by people that are not familiar with development processes, so I suggest to keep these check list stuff for the pull request template.